Assets/Project/Scripts/GamePlayScene/Gimmick/ThunderController.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections;
+using UnityEngine;
+using Project.Scripts.Utils.Definitions;
+using System.Collections.Generic;
+using System.Linq;
+using Project.Scripts.Utils.PlayerPrefsUtils;
+
+namespace Project.Scripts.GamePlayScene.Gimmick
+{
+    public class ThunderController : AbstractGimmickController
+    {
+        /// <summary>
+        /// 雲をタイルの比率でオフセットさせる
+        /// </summary>
+        private static readonly float _CLOUD_OFFSET_BY_TILE_RATIO = 0.6f;
+
+        /// <summary>
+        /// Idleステートのハッシュ値
+        /// </summary>
+        private static int IDLE_STATE_NAME_HASH = Animator.StringToHash("Idle");
+
+        /// <summary>
+        /// 雷を放つ際再生するSE
+        /// </summary>
+        [SerializeField] private AudioClip _attackSE;
+
+        /// <summary>
+        /// 移動速度
+        /// </summary>
+        [SerializeField] private float _moveSpeed = 1.5f;
+
+        /// <summary>
+        /// 目標タイル
+        /// </summary>
+        private List<(ERow row, EColumn column)> _targets;
+
+        /// <summary>
+        /// 雲オブジェクト
+        /// </summary>
+        private Transform _cloud;
+
+
+        // 各コンポーネント
+        private Animator _animator;
+        private AudioSource _audioSource;
+        private Rigidbody2D _rigidBody;
+
+        private void Awake()
+        {
+            _animator = GetComponent<Animator>();
+            _audioSource = GetComponent<AudioSource>();
+            _rigidBody = GetComponent<Rigidbody2D>();
+
+            // SE音量を適用
+            _audioSource.volume *= UserSettings.SEVolume;
+
+            // 雲をタイルの少し上に移動する
+            _cloud = transform.Find("Cloud");
+            _cloud.Translate(0, _CLOUD_OFFSET_BY_TILE_RATIO * TileSize.HEIGHT, 0, Space.Self);
```

Comment:
デフォルトが `Space.Self` だから，記述を減らせるかも
```suggestion
            _cloud.Translate(0, _CLOUD_OFFSET_BY_TILE_RATIO * TileSize.HEIGHT, 0);
```